docs/source/tasks_settings.rst

```diff
@@ -87,3 +87,18 @@ If set to true, tasks will synchronously execute when their ``enqueue()`` method
 
 Whilst not generally useful in production, this can be quite helpful for straightforward
 debugging of tasks in local environments.
+
+
+``GCP_TASKS_DISABLE_EXECUTION``
+---------------------------
```

Comment:
it's finicky, but some RST linters care if the number of dashes matches the number of characters in the previous line, so this could lead to a later changed line of code (if, for example, I opened the file).

Reply:
Made the underline match the title